test/first-order/no-arguments.coffee

```diff
@@ -64,12 +64,23 @@ module.exports = async ->
 
   run "sass"
 
+  define "handlebars", async ->
+    yield go [
+      glob "*.hb", src
```

Comment:
Well actually after looking at this, this is a test so you are fine here. However, when you get to the part where you start compiling them, you may want to use the array with file extensions above.